### PR TITLE
chore: add check for requirements.txt file

### PR DESCRIPTION
**Pull request by mpeddada1**
Add check to test `.kokoro/requirements.txt` file against Python 3.6.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -84,6 +84,13 @@ jobs:
   enforcer:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: zulu
+          java-version: 11
+      - run: java -version
+      - run: mvn enforcer:enforce@enforce -T 1C
```
Comment (suztomo) [inline .github/workflows/ci.yaml:93]: I don't think you intend to make change here

Comment (mpeddada1) [reply .github/workflows/ci.yaml]: oof thank you. This was unintentional.

**Comment by mpeddada1** [comment]
Check fails as expected:
```
ERROR: Could not find a version that satisfies the requirement protobuf==3.20.2 (from versions: 2.0.0b0, 2.0.3, 2.3.0, 2.4.1, 2.5.0, 2.6.0, 2.6.1, 3.0.0a2, 3.0.0a3, 3.0.0b1, 3.0.0b1.post1, 3.0.0b1.post2, 3.0.0b2, 3.0.0b2.post1, 3.0.0b2.post2, 3.0.0b3, 3.0.0b4, 3.0.0, 3.1.0, 3.1.0.post1, 3.2.0rc1, 3.2.0rc1.post1, 3.2.0rc2, 3.2.0, 3.3.0, 3.4.0, 3.5.0.post1, 3.5.1, 3.5.2, 3.5.2.post1, 3.6.0, 3.6.1, 3.7.0rc2, 3.7.0rc3, 3.7.0, 3.7.1, 3.8.0rc1, 3.8.0, 3.9.0rc1, 3.9.0, 3.9.1, 3.9.2, 3.10.0rc1, 3.10.0, 3.11.0rc1, 3.11.0rc2, 3.11.0, 3.11.1, 3.11.2, 3.11.3, 3.12.0rc1, 3.12.0rc2, 3.12.0, 3.12.1, 3.12.2, 3.12.4, 3.13.0rc3, 3.13.0, 3.14.0rc1, 3.14.0rc2, 3.14.0rc3, 3.14.0, 3.15.0rc1, 3.15.0rc2, 3.15.0, 3.15.1, 3.15.2, 3.15.3, 3.15.4, 3.15.5, 3.15.6, 3.15.7, 3.15.8, 3.16.0rc1, 3.16.0rc2, 3.16.0, 3.17.0rc1, 3.17.0rc2, 3.17.0, 3.17.1, 3.17.2, 3.17.3, 3.18.0rc1, 3.18.0rc2, 3.18.0, 3.18.1, 3.18.3, 3.19.0rc1, 3.19.0rc2, 3.19.0, 3.19.1, 3.19.2, 3.19.3, 3.19.4, 3.19.5, 3.19.6, 4.0.0rc1, 4.0.0rc2, 4.21.0rc1, 4.21.0rc2, 4.21.0)
ERROR: No matching distribution found for protobuf==3.20.2
Error: Process completed with exit code 1.
```

**Comment by mpeddada1** [comment]
@suztomo could you give this another look? I've just added a check for the owlbot post-processor requirements file.

**Comment by suztomo** [comment]
The requirement.txt file worked with Python 3.10.

https://pantheon.corp.google.com/cloud-build/builds/a4a0468a-d349-4106-b904-c3592bd07a47;step=1?project=repo-automation-bots

**Comment by mpeddada1** [comment]
Ah good point! In that case, the check will only apply to the kokoro jobs. We may not even need to pin the versions in owlbot-post-processor's requirements file.

**Pull request merged**